Makefile

```diff
@@ -130,7 +130,7 @@ setup/git/hooks:
 
 .PHONY: code/run
 code/run: code/gen cluster/prepare/smtp cluster/prepare/dms cluster/prepare/pagerduty setup/service_account
-	@KUBECONFIG=TMP_SA_KUBECONFIG WATCH_NAMESPACE=$(NAMESPACE) go run ./main.go
+	@KUBECONFIG=TMP_SA_KUBECONFIG WATCH_NAMESPACE=$(NAMESPACE) QUOTA=$(DEV_QUOTA) go run ./main.go
```

Comment:
In this case I would expect the installation to fallback on the secret. If the user has run make cluster/prepare they will have the secret in place. If they didn't (I think this would be unusual) but it would fall back to the env. No need to change it but just wanted to mention it.